src/neptune/attributes/series/file_series.py

```diff
@@ -72,6 +72,9 @@ def _data_to_value(self, values: Iterable, **kwargs) -> Val:
     def _is_value_type(self, value) -> bool:
         return isinstance(value, FileSeriesVal)
 
+    def _handle_stringified_value(self, value):
```

Comment:
As this method will be common for all serieses except String series, how about moving it to the abstract `Series` class?